ipatests/test_integration/test_acme.py

```diff
@@ -397,21 +407,22 @@ def test_centralize_acme_disable(self):
         assert status == 'disabled'
 
     def test_acme_pruning_no_random_serial(self):
-        """This ACME install is configured without random serial
+        """BDB install is configured without random serial
            numbers. Verify that we can't enable pruning on it.
-
-           This test is located here because by default installs
-           don't enable RSNv3.
         """
         if (tasks.get_pki_version(self.master)
            < tasks.parse_version('11.3.0')):
             raise pytest.skip("Certificate pruning is not available")
         self.master.run_command(['ipa-acme-manage', 'enable'])
-        result = self.master.run_command(
-            ['ipa-acme-manage', 'pruning', '--enable'],
-            raiseonerr=False)
-        assert result.returncode == 1
-        assert "requires random serial numbers" in result.stderr_text
+
+        # This test is only relevant with BDB backend
+        # as with LMDB, the installer now enable RSNv3 and cert pruning
+        if get_389ds_backend(self.master) == 'bdb':
+            result = self.master.run_command(
+                ['ipa-acme-manage', 'pruning', '--enable'],
+                raiseonerr=False)
+            assert result.returncode == 1
+            assert "requires random serial numbers" in result.stderr_text
```

Comment:
This will quietly skip the test for lmdb. Will that be visible anywhere as in 45 tests and 44 executed or something like that? Should it raise a pytest.skip in this case?

Reply:
The skip won't be visible, the report will look like this:
![image](https://github.com/user-attachments/assets/69184cbe-8c11-4aef-aa97-7053a05aacf2)

This way we always have acme enabled after this test, whether it was launched with BDB or LMDB.